components/buttons/FeedBackRatingButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTheme } from '@react-navigation/native';
+import React from 'react';
+import { Pressable, StyleSheet, Text } from 'react-native';
+import type { TypeRatingStates } from '../../app/(app)/feedback';
+import StyledText from '../common/StyledText';
+
+type Props = {
+  onPress: (value: number) => void;
+  rating: TypeRatingStates;
+  testID?: string;
+};
+
+const FeedBackRatingButton = ({ onPress, rating }: Props) => {
+  const { colors } = useTheme();
+  const { icon, text, value } = rating;
+
+  return (
+    <Pressable style={[styles.pressableEmoji, { backgroundColor: colors.bg }]} onPress={() => onPress(value)}>
+      <Text style={styles.FormRatingText}>{icon}</Text>
+      <StyledText font="bold" size="sm">
+        {text}
+      </StyledText>
+    </Pressable>
```

Comment:
Could you add onSelected style that will highlight the button to show current selected value. You might have to add selected props to the button.

Reply:
fixed in commit a114b0127e8f0816c1d18a0f4f4f94f97d689cc8 and afd5db0d78d3a28725dd2f046600d594044af2c2